vignettes/split_functions.Rmd

```diff
@@ -265,25 +263,25 @@ function, which accepts functions implementing different component
 behaviors and combines them into a split function which can be used in
 a layout.
 
-Splitting, or faceting, as it is done in `rtables`, can be thought of
+Splitting, or faceting as it is done in `rtables`, can be thought of
 as the combination of 3 steps:
 
 1. preprocessing - transformation of the incoming data which will be faceted
-  - e.g., dropping unused factor levels, etc
+  - e.g., dropping unused factor levels, etc.
 2. splitting - mapping the incoming data to a set of 1 or more subsets
-   representing individual facets
-3. postprocessing - operations on the facets, e.g., combining them,
+   representing individual facets.
+3. postprocessing - operations on the facets - e.g., combining them,
```

Comment:
```suggestion
3. postprocessing - operations on the facets, e.g., combining them,
```
Is not this (as it was before better so to have it similar to the other 2 points?